content/rs/administering/access-control/password-rotation.md

```diff
@@ -0,0 +1,81 @@
+---
+Title: Database Password Rotation
+description:
+weight: $weight
+alwaysopen: false
+categories: ["RS"]
+---
+To maintain secure access to your Redis Enterprise Software (RS) cluster and databases,
+most organizational security policies require that you rotate user passwords on a regular basis.
+
+For user access to the RS web UI,
+you can set a [password expiration policy]({{< relref "/rs/administering/access-control/#setting-local-user-password-expiration" >}}) that prompts the user to change their password.
+However, for database connections that rely on password authentication,
+you need to allow for authentication with the existing password while you rollout the new password to your systems.
+
+With the RS REST API, you can add additional passwords to an RS user account for authentication to the database or the Admin Console and API.
```

Comment:
In the last paragraph, we mentioned the RS web UI. Here we're talking about the Admin Console. Is there a difference between these?

Reply:
Good point. We should clean out the web UI term from the docs.